packages/core/angular/src/connection/connection.store.ts

```diff
@@ -24,11 +32,24 @@ export class ConnectionStore extends ComponentStore<ConnectionState> {
             ...this._config,
         };
 
-        this.setState({ connection: null });
+        this.setState({
+            connection: null,
+            endpoint: clusterApiUrl(WalletAdapterNetwork.Devnet),
```

Comment:
In the React version it's required and there is no default. Why does this need a default value?

Reply:
I think in this version it should behave the same as React... The default value is not needed anyway. I think there should be a type-safe way to ensure package consumers set the endpoint. (Just like in the other comment)